.github/pull_request_template.md

```diff
@@ -0,0 +1,36 @@
+## Description
+
+> Provide a brief description about **why** this PR is necessary. Be sure to provide context.
+
+## Related issues and/or PRs
+
+> If this PR addresses or references any issues and/or other PRs, list them here. For example, "Fixes #123", "Related to #456", or "Addresses #789".
+
+## Changes made
+
+> Outline the specific changes made in this pull request. Include relevant details, such as added features, bug fixes, code refactoring, or improvements.
+
+## How these changes were tested
+
+> Describe the testing that you have performed to ensure the changes are working as expected. Include details about the test cases or scenarios covered.
```

Comment:
Would you like us to write details about the unit and/or integration test cases for this section?
Reviewers can see the test cases by checking the code directly, so adding further explanation here may be redundant information.

Reply:
@Torch3333 I see. I think @feeblefakie also questioned whether we needed this section, and now I have a better understanding of why. 

Would you be OK if we removed it? 

> **Note:** Any further information about testing changes that aren't obvious from looking at the code could, of course, be added to the `Additional notes` section.

Reply:
Yes, I think we can remove it, and as you suggest, we can add further information regarding testing in the `Additional notes` section.